code/noRAM/app/src/main/java/com/example/noram/OrganizerEventListFragment.java

```diff
@@ -170,6 +171,11 @@ public void afterTextChanged(Editable editable) {
             public void onItemClick(AdapterView<?> parent, View view, int position, long id) {
                 Event event = allEventDataList.get(position);
                 // TODO: display event information with organizer page
+
+                // Temp: Edit event
```

Comment:
Is this temp and the TODO above done?

Reply:
Neither is. TODO needs to be done. Temp is done in place of the actual functionality for the moment.